Sources/PostgresNIO/New/Data/UUID+PostgresCodable.swift

```diff
@@ -1,4 +1,5 @@
 import NIOCore
+import NIOFoundationCompat
```

Comment:
why did that happen?

Reply:
It's required for `ByteBuffer.writeUUIDBytes(_:)` and `.readUUIDBytes()`. The need for the import becomes explicit when building in `BUILDING_DOCC` mode (which basically removes `Exports.swift` and thus all uses of `@_exported`). This was our workaround for the issues DocC has with re-exported imports - `@_documentation(visibility:)` didn't come along until later.